expressapi/src/routes/userRoutes.js

```diff
@@ -12,43 +18,52 @@ module.exports = (app) => {
 
   app.get("/user/:id", async (req, res) => {
     const id = parseInt(req.params.id);
-    if (isNaN(id)) {
-      res.status(400).text("Invalid user id" );
+    if (!Number.isInteger(Number(id))) {
```

Comment:
parseInt의 리턴 값은 이미 Number 입니다. 즉 `Number(id)` 는 의미가 없습니다.
parseInt의 param이 int로 바꿀 수 없는 경우에는 NaN이 리턴됩니다. 즉 해당 if문은
```suggestion
    if (isNaN(id)) {
```
으로 더 깔끔하게 바꿀 수 있습니다.